app/models/voltron/notification/sms_notification.rb

```diff
@@ -82,15 +82,15 @@ def send_now
         loop do
           break if all_attachments.count == 1
           client.messages.create({ from: from_formatted, to: to_formatted, media_url: all_attachments.shift, status_callback: callback_url }.compact)
-          @request << Rack::Utils.parse_nested_query(client.last_request.body)
-          @response << JSON.parse(client.last_response.body)
+          @request << client.http_client.last_request
+          @response << client.http_client.last_response.body
```

Comment:
These are the only things I'm not 100% are "compatible", but I don't think we use these fields anyway.